src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1642,6 +1642,10 @@ CHIP_ERROR CASESession::OnMessageReceived(ExchangeContext * ec, const PayloadHea
     Protocols::SecureChannel::MsgType msgType = static_cast<Protocols::SecureChannel::MsgType>(payloadHeader.GetMessageType());
     SuccessOrExit(err);
 
+#if CHIP_CONFIG_SLOW_CRYPTO
+    ReturnErrorOnFailure(mExchangeCtxt->SendStandaloneAckMessage());
+#endif // CHIP_CONFIG_SLOW_CRYPTO
```

Comment:
Please only send on Sigma*, not statusreport

Reply:
fixed